.github/workflows/zips.yml

```diff
@@ -223,6 +218,31 @@ jobs:
           edit-mode: replace
           body: Build files for ${{ github.event.pull_request.head.sha }} have been deleted.
 
+  dryrun-publish-to-wporg:
+    name: '[DRY RUN] Publish to WordPress.org'
+    runs-on: ubuntu-latest
+    # environment: do not define to omit SVN credentials + deployment protection rules for this dry-run.
+    if: github.event_name == 'workflow_dispatch' && ( github.ref_type == 'tag' || inputs.release_version != '' )
```

Comment:
Both of these jobs still only run on `workflow_dispatch` so there should be no issue in running this one without the approval step which is a deployment protection rule anyways.